db/migrate/20240223120822_add_unique_index_reaction_moderation_report.rb

```diff
@@ -0,0 +1,12 @@
+class AddUniqueIndexReactionModerationReport < ActiveRecord::Migration[7.0]
+  def change
+    add_index :moderation_reports,
+              %i[user_id reportable_type reportable_id],
+              unique: true,
+              name: "index_moderation_reports_on_user_and_reportable"
+    add_index :reactions,
+              %i[user_id reactionable_type reactionable_id reaction_value],
+              unique: true,
+              name: "index_reactions_on_user_and_reactionable"
```

Comment:
@shilendras 

I think this migration will crash if a DB already has duplicates which were created before this migration.

Currently, it is possible to create the duplicate reactions from UI by a user

Reply:
@shilendras updating the migration to handle this;

Reply:
Yes. Was writing this to the migration! 😄 Updated now!